pmd-core/src/main/java/net/sourceforge/pmd/cpd/AntlrTokenizer.java

```diff
@@ -53,7 +53,7 @@ protected AntlrTokenFilter getTokenFilter(final AntlrTokenManager tokenManager)
         return new AntlrTokenFilter(tokenManager);
     }
 
-    /* default */ static CharStream getCharStreamFromSourceCode(final SourceCode sourceCode) {
+    public /* default */ static CharStream getCharStreamFromSourceCode(final SourceCode sourceCode) {
```

Comment:
ok, for now we'll make it public. I'll remove the comment `/* default */`, then, because that's now wrong.
The whole class is currently marked as `@InternalApi`, so the API can be changed (probably this static helper methods is moved somewhere else...)